Mastodon/Scene/SuggestionAccount/TableViewCell/SuggestionAccountTableViewCell+ViewModel.swift

```diff
@@ -138,4 +74,62 @@ extension SuggestionAccountTableViewCell {
         .assign(to: \.isPending, on: viewModel)
         .store(in: &disposeBag)
     }
+
+    class ViewModel {
+        var disposeBag = Set<AnyCancellable>()
+        
+        @Published public var userIdentifier: UserIdentifier?       // me
+        
+        @Published var avatarImageURL: URL?
```

Comment:
Why is this `internal` whereas `authorName` and `authorUsername` are private? Shouldn't it be on the same access level?

Reply:
To be honest: I don't know (yet), I didn't write this code, I just moved it around a bit :)